OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj

```diff
@@ -23,6 +23,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Eluant">
+      <HintPath>../engine/thirdparty/download/Eluant.dll</HintPath>

```

Comment:
I doubt this is needed.

Reply:
Visual Studio will refuse to compile without it.